testcases/kernel/syscalls/symlink/symlink01.c

```diff
@@ -1879,7 +1879,7 @@ void help(void)
 
 	printf("   -T id  Determines which tests cases to execute:\n");
 
-	for (ind = 0; ind < sizeof(all_tcses) / sizeof(struct tcses); ind++) {
+	for (ind = 0; ind < (int)sizeof(all_tcses) / sizeof(struct tcses); ind++) {
```

Comment:
Cast the result => needs brackets (
`for (ind = 0; ind < (int)(sizeof(all_tcses) / sizeof(struct tcses)); ind++) {`

Although as Cyril noted, you might just use `unsigned int ind;`. I thought casting is more common, but if there is too much casting, changing type is probably better.

Reply:
I agree with both of you. In this case, I think the code is cleaner and easier to read, if we change the type to `unsigned int ind;`